dulwich/contrib/ssh_keygen_criterion.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2024 E. Castedo Ellerman <castedo@castedo.com>
+
+# Dulwich is dual-licensed under the Apache License, Version 2.0 and the GNU
+# General Public License as public by the Free Software Foundation; version 2.0
+# or (at your option) any later version. You can redistribute it and/or
+# modify it under the terms of either of these two licenses.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# You should have received a copy of the licenses; if not, see
+# <http://www.gnu.org/licenses/> for a copy of the GNU General Public License
+# and <http://www.apache.org/licenses/LICENSE-2.0> for a copy of the Apache
+# License, Version 2.0.
+# fmt: off
```

Comment:
A few reasons:

* make it so the CI checks don't fail
* not waste time bikeshedding about useless extreme esthetics of whitespace consistency
* make the code MORE readable rather than LESS readable (ruff format literally makes this code less readable)

Why do you block CI integration tests on ruff formatting? Why do you care?